Create-And-Setup-BCHE.pl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/perl
+# This version is designed to run in the server2/analysis subdirectory
+# This file creates the Database and Table on the database server
+# and prepares the analysis directory locally... 
+# updated for proteins in GMX core runs
+
+# Getting user's argument here
+use DBI;
+$input = "\n     Usage\:  Create-And-Setup-BCHE.pl \n\n";
```

Comment:
I don't think this is used...? You usually get user arguments with @ARGV as mentioned here: https://www.cyberciti.biz/faq/howto-pass-perl-command-line-arguments/

Reply:
You're right, I forgot to remove that Comment Line. We should only have the $input line, incase user add an argument on accident. Then it'll tell them how to use the script.